src/i18n/ja/nav.ts

```diff
@@ -33,6 +33,7 @@ export default NavDictionary({
 	'core-concepts/framework-components': 'UIフレームワーク',
 	'guides/server-side-rendering': 'サーバーサイドレンダリング（SSR）',
 	'guides/styling': 'CSSとスタイル',
+	'guides/cms': 'CMS',
```

Comment:
In English the label is “Connecting a CMS” rather than just “CMS”. Is there a good equivalent phrase we could use or do you think “CMS” is the best translation?